static/app/components/sidebar/index.tsx

```diff
@@ -74,12 +75,21 @@ function Sidebar({location, organization}: Props) {
     dispatchSidebar({type: 'hide panel'});
   }, [dispatchSidebar]);
 
-  const togglePanel = useCallback(
-    (panel: SidebarPanelKey) => {
-      dispatchSidebar({type: 'toggle panel', payload: panel});
-    },
-    [dispatchSidebar]
-  );
+  const showPerformanceOnboardingPanel = useCallback(() => {
```

Comment:
how do you feel about these just being part of `useSidebarDispatch`?

Reply:
I dont really mind, but I did this to avoid components that just consume the dispatch from rerendering (and vice versa) when sidebar changes